react-native-libraries.json

```diff
@@ -13862,5 +13862,14 @@
     "expoGo": true,
     "dev": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/fabrizioiacobucci/expo-googlenearby-connection",
+    "npmPkg": "expo-googlenearby-connection",
```

Comment:
```suggestion
```

`npmPkg` can be skipped in this case:
* https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg

Reply:
Done, thanks.

Reply:
Thanks for the update! 👍